airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/s3/parquet/ParquetSerializedBuffer.kt

```diff
@@ -46,7 +47,7 @@ private val logger = KotlinLogging.logger {}
  * data will be buffered in such a hadoop file.
  */
 class ParquetSerializedBuffer(
-    config: S3DestinationConfig,
+    uploadFormatConfig: UploadFormatConfig,
```

Comment:
This was the main change, decoupling ParquetBuffer from S3 bucket configs. Rest all are renames.